package/src/components/CheckoutAction/v1/CheckoutAction.md

```diff
@@ -0,0 +1,146 @@
+### CheckoutAction
+
+#### Overview
+The CheckoutActionComplete component will show when a checkout action has been completed. It is a summary of the information inside the completed action.
+
+#### Basic usage, `active` status
+```jsx
+const cart = {};
```

Comment:
for all the examples, I remove all one line variable declarations and just pass the value directly, to the component. This will make the example concise. i.e.
```
<CheckoutAction
.
.
.
cart={{}}
isLoading={false}
...
/>